### DISCUSSION TITLE
federated @connection and rover alternatives

### DISCUSSION THREAD

**Discussion by xenoterracide** [Netflix/dgs-framework#1209] [Q&A]
Apollo Rover doesn't seem to support `@connection` (it's a pretty bad tool for enterprise anyways due to lack of support for private repository mirrors and proxy issues). Does anyone have any alternatives? also how do I link this directive?shouldn't I have a statement like this for `@connection`? 

```
extend schema
  @link(
    url: "https://specs.apollo.dev/federation/v2.0"
    import: ["@key", "@external", "@extends"]
  )
```

**Comment by tmCision** [comment]
The only alternative that I am aware of is [Tyk ](https://tyk.io/blog/an-introduction-to-graphql-federation/) and it does not currently support Federation v2.
